### PR TITLE
Adjust navbar breakpoint between mobile / desktop views

### PR DESCRIPTION
**Pull request by hannahpurcell**
No ticket.  Seemed easier to just set up the new breakpoint instead of setting up a bunch of temporary classes to feature-flag this in the navbar rework.

### CODE REVIEW

File: apps/site/assets/css/core.scss [hunk 1]
```diff
@@ -13,9 +13,11 @@
 @import 'layout-core-shared';
 
 html {
+  width: 100%;
   height: 100%;
   position: relative;
   scroll-behavior: smooth;
+  overflow-x: hidden;
```
Comment (thecristen) [inline apps/site/assets/css/core.scss:20]: I'm a bit concerned about this having unintended consequences on other pages, particularly with any other elements that might have or rely on horizontal overflow/scroll behavior. 

So I'm curious what this particular change is meant to do, and whether this is the least-intrusive change we can do to accomplish it?

Comment (hannahpurcell) [reply apps/site/assets/css/core.scss:20]: Ah ok, I can omit this bit.  I noticed that some viewport sizes make the homepage have a horizontal scroll when it shouldn't.  I didn't think we had any pages on dotcom where there is horizontal scrolling -- but forgot to account for when the page is zoomed in.  Thanks for catching!

Comment (hannahpurcell) [inline apps/site/assets/css/core.scss:20]:
```suggestion
  height: 100%;
  position: relative;
  scroll-behavior: smooth;
```

File: apps/site/lib/site_web/views/event_view.ex [hunk 2]
```diff
@@ -218,7 +218,7 @@ defmodule SiteWeb.EventView do
         |> Timex.Duration.from_clock()
         |> Timex.Duration.to_string()
       rescue
-        e ->
+        _e ->
```
Comment (thecristen) [inline apps/site/lib/site_web/views/event_view.ex:221]: 😅 heh thanks!

**Review by thecristen — APPROVED** [review]
👉🏼 looks good!

**Pull request merged**